packages/edit-widgets/src/store/actions.js

```diff
@@ -160,17 +159,16 @@ export function* saveWidgetArea( widgetAreaId ) {
 				continue;
 			}
 
-			dataDispatch( 'core' ).saveEditedEntityRecord(
-				'root',
-				'widget',
-				widgetId,
-				widget
-			);
+			batchSpec.push( {
+				kind: 'root',
+				name: 'widget',
+				recordId: widgetId,
+			} );
```

Comment:
That's curious, was `saveEditedEntityRecord` being called incorrectly before? (with the last `widget` param).

Reply:
Yeah the fourth argument of `saveEditedEntityRecord` is `options` so this just did nothing.